ajax/libs/frontpress/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "frontpress",
+  "filename": "frontpress.min.js",
+  "version": "1.0.7",
+  "description": "Full front-end AngularJS template for WordPress Rest API.",
+  "keywords": [
+    "frontpress",
+    "wordpress",
+    "angular",
+    "rest",
+    "spa"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/frontpressorg/frontpress.git"
+  },
+  "author": {
+    "name": "Jota Teles",
+    "url": "https://github.com/teles"
+  },
+  "npmName": "frontpress",
+  "npmFileMap": [{
```

Comment:
Hello @teles  `[{` wrong format here. Could you run the fix tools `fixFormat.js` first https://github.com/cdnjs/cdnjs/tree/master/tools. Thanks!